### DISCUSSION TITLE
多服务器部署

### DISCUSSION THREAD

**Discussion by explorer-exe-bilibili** [Jinnrry/PMail#217] [Ideas]
我有2台服务器，一台只有ipv4，一台只有ipv6，域名的AAAA和A记录分别记录了它们的地址，它们之间通过frp连接端口，但是443端口默认被占用，怎么才能在这种特殊的情况的服务器上部署这个项目呢

**Comment by Jinnrry** [comment]
两台服务器都部署PMail，数据库使用同一个MySQL就行。把https服务关掉，这样就不需要443端口了。MX记录指向smtp.xxxx域名，smtp.xxxx域名的A记录指向你的ipv4地址，AAAA记录指向你的ipv6地址，如果发信方使用ipv6连接，你ipv6服务器就收信，如果发信方使用ipv4连接，你ipv4服务器就收信。

两台服务器使用同一个数据库，因此数据是同步的
